### DISCUSSION TITLE
NetBox turns fives years old!

### DISCUSSION THREAD

**Discussion by jeremystretch** [netbox-community/netbox#6669] [Announcements]
Five years ago, on June 27, 2016, I published the [first open source release](https://github.com/netbox-community/netbox/releases/tag/1.0.0) of NetBox. I had no idea what I was doing, nor what kind of response to expect. But thanks to the efforts of some valiant early adopters and excellent feedback from the network engineering community, NetBox quickly made a name for itself. Following an initial flurry of bug fixes and feature requests, it was clear that the project had found its place.

Looking back now, I never imagined NetBox would advance as much as it has. We've added a ton of features and functionality over the years, and we're not slowing down. Since [joining NS1](https://ns1.com/blog/netbox-network-automation-and-infrastructure-management-powerhouse) earlier this year, I've been able to renew my focus on NetBox like never before. Version 3.0 is [right around the corner](https://github.com/netbox-community/netbox/milestone/21), and we've already got the [next few 3.x releases](https://github.com/netbox-community/netbox/projects/5) planned out.

So, let me take this opportunity to once again thank the folks at [DigitalOcean](https://www.digitalocean.com/) for their early support and for enabling NetBox's open source release. And I want to thank [NS1](https://ns1.com/) for recognizing both NetBox's prevalence as a cornerstone of automation in networks today and its enormous potential going forward. But mostly I'd like to thank you, the community, and my fellow maintainers for your support in getting us to this point. NetBox would simply not be the tool that it is today without your ideas, observations, and contributions. So keep them coming, and there's no telling where the next five years will take us!

**Comment by SamuelAndradesmn4** [comment]
Congratulations netbox community and you Jeremy!!!!!

**Comment by eronlloyd** [comment]
The value of NetBox has been immense on our ability to keep network records under control. It's an essential tool for our operations. Thank you @jeremystretch for leading this project!

**Comment by joaolucasmacedo** [comment]
Congratulations. Netbox has been one of the most valuable tools to. It is becoming a standard as SoT and network documentation.
